src/ripple/core/impl/Config.cpp

```diff
@@ -162,7 +162,12 @@ void Config::setup (std::string const& strConf, bool bQuiet)
     // config directory and that with "db" as the data
     // directory.
 
-    QUIET = bQuiet;
+    QUIET = bQuiet || bSilent;
```

Comment:
There is one test where we set them for tests, so I'll write a setter. https://github.com/ximinez/rippled/blob/standalone/src/ripple/test/jtx/impl/Env.cpp#L65-L67